README.md

```diff
@@ -105,7 +105,6 @@ First, you'll need a working Ruby installation. You can install this via your op
 
 We recommend either Ruby 2.3 or the latest 2.2 version.
 
-# Creating a Sufia-based app
```

Comment:
@jcoyne Can you also update the TOC above (around line 29) to update its structure?